mobile/test/common/integration/client_integration_test.cc

```diff
@@ -391,13 +400,27 @@ void ClientIntegrationTest::explicitFlowControlWithCancels(uint32_t body_size) {
     } else {
       stream->readData(100);
     }
+
+    if (terminate_engine && request_for_engine_termination == i) {
+      {
```

Comment:
nit: Do we need these '{}'s?

Reply:
good point, we don't, I'll send a PR now to fix this.